roles/cups/templates/cups.conf.j2

```diff
@@ -21,7 +21,7 @@ location ~ ^/print(|/.*)$ {    # '~' -> '~*' for case-insensitive regex
         return 301 http://localhost:631;
     }
 
-    return 301 http://$host:631;   # For 192.168.0.x, 172.18.96.1, 10.8.0.y ETC
+    return 301 http://$host:631;   # For {{ lan_ip }}, 172.18.96.1, 10.8.0.y ETC
```

Comment:
@holta likes to use the part to the right of the # as notes, changed wrong ip address